### PR TITLE
Upgrade lerna to v3, fixes #864

### LINKED ISSUE #864: Use lerna@3
Emotions packages doesn't contain the LICENSE file when they are published to npm. This is a bug in lerna@2, and lerna@3 fixed it. Can we upgrade to it and publish them again please? Some corporate companies are forbidden to use the packages until this is fixed.

- `emotion` version: next

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

**Comment by pksunkara** [comment]
@mitchellhamilton @Andarist Here's the PR which should fix the LICENSE issues.

File: lerna.json [hunk 1]
```diff
@@ -10,7 +9,8 @@
       "Tag: Internal": ":house: Internal"
     }
   },
-  "version": "9.2.10",
   "npmClient": "yarn",
-  "useWorkspaces": true
+  "useWorkspaces": true,
+  "registry": "https://registry.npmjs.org",
+  "version": "9.2.10",
```
Comment (Ailrun) [inline lerna.json]: Wrong trailing `,`.

File: lerna.json [hunk 2]
```diff
@@ -10,7 +9,8 @@
       "Tag: Internal": ":house: Internal"
     }
   },
-  "version": "9.2.10",
   "npmClient": "yarn",
-  "useWorkspaces": true
+  "useWorkspaces": true,
+  "registry": "https://registry.npmjs.org",
```
Comment (Ailrun) [inline lerna.json]: Do we need this registry?

Comment (pksunkara) [reply lerna.json]: Yes, because yarn defaults to `yarnpkg` when publishing.

Comment (pksunkara) [reply lerna.json]: Even though we are using `npm` when publishing, it's better safe to have this.

**Comment by pksunkara** [comment]
Addressed the review and tests are passing.

**Comment by pksunkara** [comment]
Please publish packages (I only care about next-packages) once this is merged.

**Comment by tkh44** [comment]
I'd like @mitchellhamilton to check this over before merging, but lgtm.

**Comment by mitchellhamilton** [comment]
I’m fine with this but I don’t see how it would solve the LICENSE issue?

**Comment by pksunkara** [comment]
Lerna@2 when publishing packages in a monorepo doesn't do anything, thus the packages won't contain the LICENSE because it's generally in the root of the repo. Lerna@3 copies the LICENSE before preparing for publishing.

**Comment by mitchellhamilton** [comment]
Wouldn’t we have to add the LICENSE file to each package and add it to the package.json files array?

**Comment by pksunkara** [comment]
Not sure about the package.json files, but lerna will take of copying the file to each package.

**Comment by mitchellhamilton** [comment]
[It looks like LICENSE files are always published](https://docs.npmjs.com/files/package.json#files) so lgtm

Thanks!!

**Comment by tkh44** [comment]
Thanks @pksunkara and @mitchellhamilton

**Pull request merged**